packages/es-components/src/components/containers/prompt/Prompt.md

```diff
@@ -24,18 +24,19 @@ function getDate() {
 
 <React.Fragment>
   <Prompt isContentReadAloud={true}>
-    <Control orientation='inline'>
-      <Fieldset legendContent={`My name is Mr/Mrs. Agent. Today is ${getDate()}. This call will be recorded. Do I have your permission to record this conversation to confirm your enrollment?`}>
-        <RadioGroup
-          name='recordConversation'
-          onChange={radioOnChange}
-          selectedValue={recordConversation}
-        >
-        <RadioButton value='recordConversationYes'>Yes</RadioButton>
-        <RadioButton value='recordConversationNo'>No</RadioButton>
-        </RadioGroup>
-      </Fieldset>
-    </Control>
+    <div>
+      {`My name is Mr/Mrs. Agent. Today is ${getDate()}. This call will be recorded. Do I have your permission to record this conversation to confirm your enrollment?`}
```

Comment:
Do we need to use `formatMessage` with the text in this component?

Reply:
Yes, I can add that. Thanks.